packages/amplify_core/android/src/main/kotlin/com/amazonaws/amplify/amplify_core/exception/ExceptionMessages.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazonaws.amplify.amplify_core.exception
+
+class ExceptionMessages {
+    companion object {
+
+        @JvmStatic
+        val createGithubIssueString: String =
+                """
+        Please take a look at
+        https://github.com/aws-amplify/amplify-flutter/issues to see if there are any existing issues that
+        match your scenario, and file an issue with the details of the bug if there isn't.
```

Comment:
Please take a look at https://github.com/aws-amplify/amplify-flutter/issues to see if there are any existing issues that match your scenario, and feel free to comment on any exiting issues you find. If your scenario is not described in an existing issue, please file create one with a detailed desription of the bug.

Reply:
I matched it with https://github.com/aws-amplify/amplify-ios/blob/e2f7f64e2a501f3b2807481ca921e71710c82b22/Amplify/Core/Support/AmplifyErrorMessages.swift#L14 to avoid a lot of different message strings